events/bingo/webApps/bingo.js

```diff
@@ -0,0 +1,415 @@
+//
+// bingo.js
+// NodeJS Web App for Bingo
+// Created by Zach Fox on 2019-02-12
+// Copyright High Fidelity 2019
+//
+// Licensed under the Apache 2.0 License
+// See accompanying license file or http://apache.org/
+//
+//
+// NOTE 2019-02-13:
+// In its current form, this script has been closely adapted from
+// the Google Script that the Experiences team used for previous Bingo
+// events. I made a couple of bugfixes and optimizations, but not many.
+// I wanted to maintain as much backwards compatibility as possible.
+//
+
+var http = require('http');
+var url = require('url');
+var dbInfo = require('./dbInfo.json');
+
+var currentTableName;
+function createNewTable() {
+    currentTableName = Date.now();
+
+    var query = `CREATE TABLE \`${currentTableName}\` (
+        username VARCHAR(40) PRIMARY KEY,
+        cardNumbers VARCHAR(300),
+        cardColor VARCHAR(50),
+        prizeWon VARCHAR(40)
+    )`;
+    connection.query(query, function(error, results, fields) {

```

Comment:
not necessary, doesn't hurt anything. not a logical error and isn't wrong. it might be slightly cleaner to remove, but i am not worried about it in this case.